src/iam.ts

```diff
@@ -549,6 +557,34 @@ export class IAM extends IAMBase {
         throw new Error(ERROR_MESSAGES.PROVIDER_NOT_INITIALIZED);
     }
 
+    /**
+     * createIdentityProofWithDelegate
+     *
+     * @description create a raw identity proof for a delegate
+     * @returns JWT token of created identity
+     *
+     */
+    async createIdentityProofWithDelegate(
```

Comment:
I think the such signature would be accurate and supplementary to method above
```suggestion
    async createDelegateProof(
```